app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/item/VaultItemViewModel.kt

```diff
@@ -520,6 +521,21 @@ class VaultItemViewModel @Inject constructor(
         }
     }
 
+    private fun handleCopyNotesClick() {
+        onContent { content ->
+            val notes = requireNotNull(content.common.notes)
```

Comment:
We want to try to avoid stuff like this, can we add a helper function similar to the others for this:

```kotlin
    private inline fun onSecureNoteContent(
        crossinline block: (
            VaultItemState.ViewState.Content,
            VaultItemState.ViewState.Content.ItemType.SecureNote,
        ) -> Unit,
    ) {
        state.viewState.asContentOrNull()?.let { content ->
            (content.type as? VaultItemState.ViewState.Content.ItemType.SecureNote)
                ?.let { secureNoteContent -> block(content, secureNoteContent) }
        }
    }
```

Reply:
the note is not specific for the SecureNote but part of the `common`. I believe we can't apply it like this?

Reply:
Ahh, I see now. That is my mistake.

But this like could crash then:

```kotlin
val notes = requireNotNull(content.common.notes)
```

Seems like we could just do this:
```koltin
val notes = content.common.notes.orEmpty()
```